website/docs/r/linux_web_app_slot.html.markdown

```diff
@@ -162,9 +162,7 @@ An `application_stack` block supports the following:
 
 ~> **NOTE:** 10.x versions have been/are being deprecated so may cease to work for new resources in the future and may be removed from the provider.
 
-* `php_version` - (Optional) The version of PHP to run. Possible values are `7.4`, `8.0` and `8.1`.
-
-~> **NOTE:** versions `5.6` and `7.2` are deprecated and will be removed from the provider in a future version.
+* `php_version` - (Optional) The version of PHP to run. Possible values are `8.0`, `8.1` and `8.2`.
```

Comment:
as above, we need to keep 7.4 available, can you update the note to the effect:
```
~> **NOTE:** version `7.4` is deprecated and will be removed from the provider in a future version.
```

Reply:
done